frontend/src/components/DisplayProjects/index.tsx

```diff
@@ -18,7 +20,7 @@ function renderRow(props: ListChildComponentProps) {
   const { index, style } = props;
 
   return (
-    <ListItem style={style} key={index} component="a" disablePadding>
+    <ListItem style={style} key={index} component="a" disablePadding href="/project">

```

Comment:
should not be using href tags, they mess with the router (https://mui.com/material-ui/guides/routing/)